Mathlib/CategoryTheory/Yoneda.lean

```diff
@@ -436,6 +531,18 @@ theorem coreprW_hom_app (F : C ⥤ Type v₁) [F.IsCorepresentable] (X : C) (f :
 
 end Corepresentable
 
+lemma isRepresentable_comp_uliftFunctor_iff {F : Cᵒᵖ ⥤ Type v} :
+    (F ⋙ uliftFunctor.{w}).IsRepresentable ↔ F.IsRepresentable := by
+  refine ⟨fun ⟨X, ⟨R⟩⟩ ↦ ?_, fun ⟨X, ⟨R⟩⟩ ↦ ?_⟩
+  · exact ⟨X, ⟨representableByUliftFunctorEquiv R⟩⟩
+  · exact ⟨X, ⟨representableByUliftFunctorEquiv.symm R⟩⟩
```

Comment:
Purely aesthetic nitpicking, so feel free to ignore, but a combination of solely `by` refine and `exact` feels a bit weird:
```suggestion
    (F ⋙ uliftFunctor.{w}).IsRepresentable ↔ F.IsRepresentable where
  mp := fun ⟨X, ⟨R⟩⟩ ↦ ⟨X, ⟨representableByUliftFunctorEquiv R⟩⟩
  mpr := fun ⟨X, ⟨R⟩⟩ ↦ ⟨X, ⟨representableByUliftFunctorEquiv.symm R⟩⟩
```

```suggestion
    (F ⋙ uliftFunctor.{w}).IsRepresentable ↔ F.IsRepresentable where
  mp | ⟨X, ⟨R⟩⟩ => ⟨X, ⟨representableByUliftFunctorEquiv R⟩⟩
  mpr | ⟨X, ⟨R⟩⟩ => ⟨X, ⟨representableByUliftFunctorEquiv.symm R⟩⟩
```
is also an option, but is arguably more "niche" syntax.

Same below.